pkg/graph/schema/types/mto_common_solution_system_owner.graphql

```diff
@@ -0,0 +1,53 @@
+type MTOCommonSolutionSystemOwner {
+  id: UUID!
+  key: MTOCommonSolutionKey!
+
+  ownerType: MTOCommonSolutionOwnerType!
+  cmsComponent: String!
+
+  createdBy: UUID!
+  createdByUserAccount: UserAccount!
+  createdDts: Time!
+  modifiedBy: UUID
+  modifiedByUserAccount: UserAccount
+  modifiedDts: Time
+}
+
+"""
+Enum for owner type
+"""
+enum MTOCommonSolutionOwnerType {
+  BUSINESS_OWNER
+  SYSTEM_OWNER
+}
+
+"""
+Input for creating or updating a common solution system owner
+"""
+input MTOCommonSolutionSystemOwnerChanges @goModel(model: "map[string]interface{}") {
+  ownerType: MTOCommonSolutionOwnerType!
+  cmsComponent: String!
+}
+
+extend type Query {
+  mtoCommonSolutionSystemOwner(id: UUID!): MTOCommonSolutionSystemOwner!
+    @hasAnyRole(roles: [MINT_USER, MINT_ASSESSMENT])
+}
+
+extend type Mutation {
+  createMTOCommonSolutionSystemOwner(
+    key: MTOCommonSolutionKey!
+    ownerType: MTOCommonSolutionOwnerType!
+    cmsComponent: String!
```

Comment:
You can reuse the changes type here. Var name could be different (I think we have this paradigm somewhere else in the app.) What do you think
```suggestion
    changes: MTOCommonSolutionSystemOwnerChanges!
```

Reply:
sounds good, we have it so can reuse